src/style.css

```
@@ -1 +1,35 @@
-/* add styles here */
+.star {
+  padding: 22px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('./images/star.png');
+  width: 16px;
+  height: 16px;
+  margin: 4px;
+  background-size: cover;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url('./images/gold-star.png');
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url('./images/gold-star.png');
+}
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2) {
  background-image: url('./images/gold-star.png');
}
```
you can combine all those selectors like that